src/helpers/discord_utils.ts

```diff
@@ -207,7 +208,8 @@ export async function sendOptionsMessage(message: Eris.Message<Eris.GuildTextabl
 
     await sendInfoMessage(getMessageContext(message),
         updatedOption === null ? "Options" : `${updatedOption.option} ${updatedOption.reset ? "reset" : "updated"}`,
-        `Now playing the ${optionStrings[GameOption.LIMIT]} out of the __${totalSongs}__ most popular songs by ${guildPreference.isGroupsMode() ? optionStrings[GameOption.GROUPS] : optionStrings[GameOption.GENDER]} ${optionStrings[GameOption.CUTOFF]}\
+        `Now playing the ${optionStrings[GameOption.LIMIT]} out of the __${totalSongs}__ most popular songs by ${guildPreference.isGroupsMode() ? optionStrings[GameOption.GROUPS] : optionStrings[GameOption.GENDER]}\
+        ${guildPreference.isGroupsMode() && guildPreference.getGender()[0] === GENDER.ALTERNATING && guildPreference.getGroupIds().length > 1 ? ` with ${optionStrings[GameOption.GENDER]}` : ""} ${optionStrings[GameOption.CUTOFF]}\
```

Comment:
`guildPreference.getGroupIds().length > 1` => `isGroupsMode()`?

Reply:
When only 1 group is selected, I hide the `alternating` portion. I can change this if you want